content/en/docs/concepts/services-networking/connect-applications-service.md

```diff
@@ -134,6 +134,10 @@ about the [service proxy](/docs/concepts/services-networking/service/#virtual-ip
 Kubernetes supports 2 primary modes of finding a Service - environment variables
 and DNS. The former works out of the box while the latter requires the
 [CoreDNS cluster addon](http://releases.k8s.io/{{< param "githubbranch" >}}/cluster/addons/dns/coredns).
+{{< note >}}
+If the service environment variables are not desired (because possible clashing with expected program ones, too many variables to process, only using DNS, etc) this mode can be disabled setting the `enableServiceLinks` flag to `false` on the [pod spec](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#pod-v1-core). 
```

Comment:
```suggestion
If the service environment variables are not desired (because possible clashing with expected program ones,
too many variables to process, only using DNS, etc) this mode can be disabled by setting the `enableServiceLinks` 
flag to `false` on the [pod spec](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#pod-v1-core). 
```

Reply:
setting -> by setting.

Reply:
If you're revising this @eroldan, also consider switching the text into an active voice.

One example, to illustrate the sort of change I mean: “…you can disable this mode: change the pod spec to set the flag `enableServiceLinks` to false.”